src/test/cpp/main.cpp

```diff
@@ -1,3 +1,7 @@
+#include "utils/ConsoleLogger.h"
+
 int main(int argc, char** argv) {
+  auto& instance = ConsoleLogger::getInstance();
```

Comment:
We should remove / comment out this test before merging

Reply:
why